assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -16,8 +16,10 @@ const dimensions = {
     intcmp: 'dimension12'
 };
 
+const defaultTracker = 'membershipPropertyTracker';
+var gaEnabled = true;
```

Comment:
Hmm, maybe initialise an no op function in ga.es6 so that if it's called without being initialised then it doesn't do anything. Rather than storing another proxy for the do not track  status. I can backport this into the membership ga js. (which is the same)